internal/blueprint/blueprint_test.go

```diff
@@ -89,3 +90,26 @@ func TestBumpVersion(t *testing.T) {
 		}
 	}
 }
+
+func TestGetPackages(t *testing.T) {
+    
+	bp := Blueprint{
+		Name:        "packages-test",
+		Description: "Testing GetPackages function",
+		Version:     "0.0.1",
+		Packages: []Package{
+			{Name: "tmux", Version: "1.2"}},
+        Modules: []Package{
+            {Name: "openssh-server", Version: "8.*"}},
+        Groups: []Group{
+            {Name: "anaconda-tools"}},
+
+	}
+    out := bp.GetPackages()
```

Comment:
Other than naming this `result` looks good.

Reply:
Sorry, forgot to change this, this was just a placeholder, thanks for noticing.